ci/scripts/update-manifest

```diff
@@ -5,13 +5,13 @@ RELEASE_NAME=$2
 VERSION=$3
 SHA1=$4
 MANIFEST_PATH=$5
-: ${MANIFEST_PATH:?USAGE: ./ci/scripts/update-manifest GITHUB_OWNER RELEASE_NAME VERSION SHA1 MANIFEST_PATH}
+: "${MANIFEST_PATH:?USAGE: ./ci/scripts/update-manifest GITHUB_OWNER RELEASE_NAME VERSION SHA1 MANIFEST_PATH}"
 
 set -e -u
 
-manifest_len=$(wc -l $MANIFEST_PATH | awk '{print $1}')
-manifest_head=$(head -n `expr $manifest_len - 4` $MANIFEST_PATH)
-cat > $MANIFEST_PATH <<YAML
+manifest_len=$(wc -l "$MANIFEST_PATH" | awk '{print $1}')
+manifest_head=$(head -n "$(( manifest_len - 4))" "$MANIFEST_PATH")
```

Comment:
just to get my head around this... it will trim the last 4 lines off of manifest_path?

head accepts negative counts, which just deduct from the end:
```shell
echo -e "1\n2\n3\n4\n5\n6" | head -n -4
1
2
```

Reply:
Good point, fixed now.